ModManagerUI/ModManagerUIPlugin.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BepInEx;
+using BepInEx.Logging;
+using HarmonyLib;
+using ModManager.ModIoSystem;
+using ModManager.StartupSystem;
+using ModManagerUI.LocalizationSystem;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ModManagerUI
+{
+    [BepInPlugin("com.modmanagerui", "Mod Manager UI", "0.1.0")]
+    public class ModManagerUIPlugin : BaseUnityPlugin
+    {
+        public static ManualLogSource Log;
+
+        public void Awake()
+        {
+            Log = Logger;
+            // Add ModManagerStartup.Run() here
```

Comment:
here ? but not anymore?

Reply:
This is a placeholder to remind that the ModManagerStartup.Run needs to be added. Atm locally my values are hardcoded stuff which shouldn't even go to version control